docs/_source/docs/stack_config.rst

```diff
@@ -205,6 +206,22 @@ Examples include:
 
 ``on_failure: "DELETE"``
 
+disable_rollback
+~~~~~~~~~~~~~~~~
+* Resolvable: No
+* Can be inherited from StackGroup: Yes
+* Inheritance strategy: Overrides parent if set
+
+This parameter describes the action taken by CloudFormation when a Stack fails
+to create or update, default is False. This option can be set from the stack
+config or from the Sceptre CLI commands to deploy stacks. Enabling disable-rollback
+using the CLI option disables rollback globally for all stacks. This option
+overrides on_failure since Cloudformation does not allow setting both on deployment.
+For more information and valid values see the `AWS Documentation`_.
+
+Examples include:
+
+``disable_rollback: "True"``
```

Comment:
Why make this a stack config?

Reply:
I kind of feel like this should _only_ be a cli flag and not a stack config at all. What's the use case of having this as a stack configuration?

Reply:
good question,  it’s because i’ve followed the pattern that’s been setup in the existing code base.  it’s actually what you suggested in [your comment on the issue](https://github.com/Sceptre/sceptre/issues/1179#issuecomment-997427832).  We implement CLI flag only in the code or just leave this PR as-is and document CLI flag without documenting the `disable-rollback` stack config.

Reply:
kept as-is since there seems to be some valid use cases for setting disable_rollback in stack config